cmasher/utils.py

```diff
@@ -233,6 +239,117 @@ def _get_cmap_perceptual_rank(
 
 
 # %% FUNCTIONS
+# This function combines multiple colormaps at given nodes
+def combine_cmaps(
+    cmaps: list[Union[Colormap, str]],
+    *,
```

Comment:
In this round of review, I'll make suggestions to change the interface so that positional arguments are used for cmaps instead of a list as the first argument. This is the fundamental change, all following suggestions will be adapting call sites and docs
```suggestion
    *cmaps: Union[Colormap, str],
```

Reply:
I think I'm not understanding this change well enough here (especially the pros and cons)... 

In current implementation, `cmaps` would take exactly a `list`; while in the suggested implementation, any number of positional args would be recognised as parts of `cmaps` and would be zipped during runtime.

From my understanding, the current implementation is "stricter" by explicitly requiring a `list` of `Colormaps`, and suggested implementation being more flexible ("lighter" as you described).

By making this change (and the other change to make all other args keyword), we are being more flexible by requesting other args to be keyword, leaving only `cmaps` positional, am I right? 

I feel this should be safe and good as this utility would only take several `Colormaps` and combine them, so it's reasonable to expect `cmaps` to be the focus here.